src/pages/DistributionsPage/AllocationsTable.tsx

```diff
@@ -158,34 +158,28 @@ export const AllocationsTable = ({
               {numberWithCommas(givenPercent(user.received) * 100, 2)}%
             </td>
             <td className="alignRight">
-              {user.circle_claimed
-                ? `${smartRounding(
-                    circleDist &&
-                      circleDist.distribution_type ===
-                        DISTRIBUTION_TYPE.COMBINED
-                      ? user.circle_claimed - user.fixed_payment_amount
-                      : user.circle_claimed
-                  )} ${tokenName || 'GIVE'}`
+              {circleDist
+                ? `${smartRounding(user.circleClaimed)} ${tokenName || 'GIVE'}`
```

Comment:
In order to be truly precise, this would mean refactoring `smartRounding` to use ethers [formatUnits](https://docs.ethers.io/ethers.js/v3.0/html/api-utils.html#ether-strings-and-wei), but I think it'll minimize tech debt later.